custom_components/ha_washdata/ws_api.py

```
@@ -3433,6 +3440,38 @@ async def ws_clear_suggestions(
+    manager = _get_manager(hass, entry_id)
+    if manager is None:
+        _err_not_found(connection, msg["id"], entry_id)
+        return
+    try:
+        await manager.profile_store.set_suggestion_locked(msg["key"], bool(msg["locked"]))
+        manager.notify_update()
+        _send_result(
+            connection, msg["id"], "set_suggestion_lock",
+            {"success": True, "locked_suggestions": manager.profile_store.get_locked_suggestions()},
+        )
+    except Exception as exc:  # pylint: disable=broad-exception-caught
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)